docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -2,14 +2,18 @@
 
 = Adopting Compute services to the {rhos_acro} data plane
 
-.Prerequisites
+Adopt your Compute (nova) services to the {rhos_long} data plane.
+
+//kgilliga: The following text belongs under the code block in step 6 but I'm unable to hide it there: "For multi-cell, config maps and {rhos_prev_long} data plane services should be named like `nova-custom-ceph-cellX` and `nova-compute-extraconfig-cellX`."
 
-* Remaining source cloud xref:stopping-infrastructure-management-and-compute-services_{context}[Stopping infrastructure management and Compute services] on Compute hosts.
-* Ceph backend for Nova/Libvirt is configured xref:configuring-a-ceph-backend_migrating-databases[Configuring a Ceph backend].
-* Make sure the IPAM is configured
```

Comment:
IPAM is IP Address Management? Missing a full stop / period at the end of this sentence.
You could drop "make sure" to make it minimal. 
By "configured", do you mean "configured and deployed"? I'm thinking that something could be configured but still "switched off" kind of situation. 
I guess these lines have been removed anyway.